sdk/php/composer.json

```diff
@@ -29,7 +29,7 @@
     },
     "require-dev": {
         "phpunit/phpunit": "^11.0",
-        "symfony/console": "^6.3|^7.0",
```

Comment:
`symfony/console` has been removed from `require-dev` because it is already in the `require` section, making this line unnecessary as the package is already required.